Nodejs/Setup/BuildRelease.ps1

```diff
@@ -673,9 +706,12 @@ try {
                 } } | `
                 %{ Copy-Item $_.src $_.dest -Force -EA 0; $_ } | `
                 %{ "Copied $($_.src) -> $($_.dest)" }
+
+            $vsmanFile = "$($outdir)\NodejsTools.vsman"
+            fix-vs-manifest $vsmanFile $i.final_msidir
```

Comment:
Per the build errors, this should be conditional on VsTarget == "15.0", as the files won't exist for Dev14 builds.